### PR TITLE
Get proposal id with events

### PR DESCRIPTION
**Pull request by Techget**
- Get proposal id with events to avoid problems when using RPC node does not support `debug_traceTransaction`
- Import ERC20Votes in GovernorContract for build

### CODE REVIEW

File: scripts/governance_standard/deploy_and_run.py [hunk 1]
```diff
@@ -113,15 +113,17 @@ def propose(store_value):
         tx = account.transfer(accounts[0], "0 ether")
         tx.wait(1)
     propose_tx.wait(2)  # We wait 2 blocks to include the voting delay
-    # This will return the proposal ID
-    print(f"Proposal state {GovernorContract[-1].state(propose_tx.return_value)}")
+    # This will return the proposal ID, brownie.exceptions.EventLookupError will be 
+    # thrown if ProposalCreated event is not emitted.
+    proposal_id = propose_tx.events['ProposalCreated']['proposalId']
```
Comment (PatrickAlphaC) [inline scripts/governance_standard/deploy_and_run.py]:
```suggestion
    proposal_id = propose_tx.events['ProposalCreated']['proposalId'] # you could also do `propose_tx.return_value` if your node allows
```

File: contracts/governance_standard/GovernorContract.sol [hunk 2]
```diff
@@ -6,6 +6,7 @@ import "@openzeppelin/contracts/governance/extensions/GovernorCountingSimple.sol
 import "@openzeppelin/contracts/governance/extensions/GovernorVotes.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorVotesQuorumFraction.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
```
Comment (PatrickAlphaC) [inline contracts/governance_standard/GovernorContract.sol]: What's the purpose of this change?

**Review by PatrickAlphaC — CHANGES_REQUESTED** [review]

**Comment by PatrickAlphaC** [comment]
Thanks for this PR! Just one question...

File: contracts/governance_standard/GovernorContract.sol (hunk 2 above)
Comment (Techget) [reply contracts/governance_standard/GovernorContract.sol]: Previously, received error as 

```
CompilerError: solc returned the following errors:

DeclarationError: Identifier not found or not unique.
  --> contracts/governance_standard/GovernorContract.sol:23:9:
   |
23 |         ERC20Votes _token,
   |         ^^^^^^^^^^

      ^^^^^^^^^^
```

But I just tried to repro with new a fresh `brownie bake dao-mix`, this error stopped happening, will remove it from this PR.

**Comment by Techget** [comment]
jfyi, have updated the pull requests

**Review by PatrickAlphaC — APPROVED** [review]

**Comment by PatrickAlphaC** [comment]
Awesome, thank you!!
